libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.spec.ts

```diff
@@ -26,13 +26,16 @@ import { DhAdminUserManagementDataAccessApiStore } from '@energinet-datahub/dh/a
 
 import { DhUsersTabComponent } from './dh-users-tab.component';
 import { searchDebounceTimeMs } from './dh-users-tab-search.component';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatSelectHarness } from '@angular/material/select/testing';
 
-describe(DhUsersTabComponent.name, () => {
+describe('DhUsersTabComponent.name', () => {
```

Comment:
Suggestion: Drop .name :-)

Reply:
Gonna drop quotes. :P

Reply:
... wait, you can drop name. Why are we typing name then, at all.

Reply:
I's mainly for making sure the value always reflects the class, also in case the class gets renamed :)